app/dev-portal/portals/customization/snippets.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Snippets
+---
+
+Snippets enable reuse of content and MDC components across [Pages](/dev-portal/portals/customization/custom-pages). Visibility controls and Publishing status allow you to enable/disable Snippets, and/or restrict access to logged-in Developers.
+
+To get started creating Snippets, navigate to your Dev Portal and select [**Portal Editor**](/dev-portal/portals/customization/portal-editor/), then Snippets.
+
+{:.note}
+> *Snippets are built using Markdown Components (MDC). Additional documentation on Snippet syntax and usage is available on a [dedicated MDC site](https://portaldocs.konghq.com/components/snippet).*
+
+
+### Create a new Snippet
+* Click to create a new Snippet at the top of the left sidebar.
+* Give the Snippet a "name". This will be used to refer to your Snippet from Pages, and must be a unique lowercase, `kebab-case` string.
+* The Snippet will be created with the `title` in front matter set to the specified name. Snippet front matter is not used in generating Pages, but can be useful to keep track of what you're working on alongside other Editors.
```

Comment:
```suggestion
1. In the Dev Portal editor sidebar, click to create a new Snippet.
1. Give the Snippet a name. This will be used to refer to your Snippet from Pages, and must be a unique lowercase string, for example: `my-snippet`.
1. The Snippet will be created with the `title` in the front matter set to the specified name. The Snippet front matter isn't used in generating Pages, but can be useful to keep track of what you're working on alongside other editors.
```